packages/ddp-server/livedata_server.js

```diff
@@ -784,20 +811,21 @@ Object.assign(Session.prototype, {
         id: msg.id
       };
 
-      promise.then((result) => {
+      try {
+        const result = Promise.await(promise);
```

Comment:
This is a big change - previously, this promise ran in the background, now this function blocks waiting for it.
1. Is it expected?
2. Is it needed?

Reply:
This change is not needed!

I was doing some tests around this, and I ended up leaving this there without thinking about the impact. 

But it's fixed now. Thanks.